.github/pull_request_template.md

```diff
@@ -9,6 +9,7 @@ Description of changes
 
 #### Merge checklist
 - [ ] Changed base branch to release branch
+- [ ] Add or update TypeScript definitions (`index.d.ts`) if necessary
```

Comment:
I added this little reminder to ensure the TypeScript definitions stay up to date, even when new components are added or old ones changed—happy to remove that again if you think it's unnecessary